api/src/controllers/TodoController/index.ts

```diff
@@ -32,12 +33,12 @@ import { logger } from '../../helpers'
  *         description: Internal server error
  */
 const get = async (req: Request, res: Response) => {
-  pool.query('SELECT * FROM todos', (err, rows) => {
-    if (err) throw err
-    logger.log('Data received todos db: test-2', rows)
-
+  try {
+    const rows = await db.select().from(todos)
```

Comment:
Move all selects/inserts into the ToDoRepository file. 
So we each entity/model we will have Repository, where we can write queries/custom queries if needs.
Path: src/repositories/ToDoRepository

Move call to repository from controller to service layer.
Path: src/services/ToDoService

The logic is: Controller -> Service -> Repo
No business logic is in controller.
All business logic is in services.

Request validation - in the controllers.
Business validation - in the services